R/apa_print_htest.R

```diff
@@ -153,6 +153,8 @@ apa_print.htest <- function(
       stop("Please provide the sample size to report.")
     # } else {
     #   n <- paste0(", n = ", n)
+    } else {
+      attr(x$statistic, "n") <- as.integer(n)
```

Comment:
Because `x` is the already-typeset `apa_results_table`, I think it would be more consistent to add the *n* as a character via `attr(x$statistic, "n") <- printnum(n)`. This would also ensure that for large *n*, the big mark is properly applied.